File: 3-Leccion-Condicionales-Ciclos/D.Imprime-pares-en-rango.cpp
#include <iostream>
using namespace std;

int main() {
  int inferior, superior;
  cin >> inferior >> superior;

  if (inferior % 2) 
    inferior++;
  
  if (superior % 2)
    superior--;

  for (int i = inferior; i <= superior; i += 2)
    cout << i << endl;

  return 0;
}

File: 3-Leccion-Condicionales-Ciclos/D.Imprime-pares-en-rango.java
import java.util.Scanner;

public class Main {
  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);

    int inferior = scanner.nextInt();
    int superior = scanner.nextInt();

    if (inferior % 2 == 1) 
      inferior++;
    
    if (superior % 2 == 1)
      superior--;

    for (int i = inferior; i <= superior; i += 2)
      System.out.println(i);
  }
}